cypress/helpers/customTypes.js

```diff
@@ -33,30 +32,49 @@ export function renameCustomType(id, actualName, newName) {
     `/ ${newName} - Edited`
   );
   cy.readFile(TYPES_FILE).should("contains", `${newName} - Edited`);
+  cy.readFile(CUSTOM_TYPE_MODEL(id)).then((model) => {
+    expect(JSON.stringify(model)).to.contain(newName);
+  });
 }
 
-export function addFieldToCustomType(
-  id,
-  fieldType,
-  fieldName,
-  fieldId,
-  isFirstField /* boolean | undefined */
-) {
-  const addFieldButtonSelector = isFirstField
-    ? "empty-zone-add-new-field"
-    : "add-new-field";
-
-  cy.get(`[data-cy="${addFieldButtonSelector}"]`).first().click();
+export function addFieldToCustomType(fieldType, fieldName, fieldId) {
```

Comment:
Are there restriction to when this can be used? ie needs to be on a specific page, or some seed data? If so would be nice to have then explained in a comment

Reply:
It needs to be done from the builder directly.

I didn't added the navigation and save actions because we usually adds more than 1 field before saving like a user would do.